app/forms/user_phone_form.rb

```diff
@@ -23,7 +23,7 @@ def initialize(user, phone_configuration)
   def submit(params)
     ingest_submitted_params(params)
 
-    success = valid?
+    success = phone_configuration || valid?
```

Comment:
How does having a phone configuration present make the form submission valid?

Reply:
That validation is only for the phone number. If a phone configuration already exists (validated by the id in & ownership of the user) it's already been validated.

Reply:
Hmm, this makes me think this form is trying to do too many things. What if we broke the new functionality off into something like `edit_phone_form` or something like that?

Reply:
Sure. I'm generally not that happy with this relationship between the controller, form, and validator anyway.